governance/src/main/java/org/apache/servicecomb/governance/marker/TrafficMarker.java

```diff
@@ -23,6 +23,10 @@
 public class TrafficMarker extends Configurable {
   private List<Matcher> matches;
 
+  private String profileValues;
+
+  private String profileExtractClass;
```

Comment:
I think it is not a good idea to create these two properties here. We need to keep the specification clear and clarity, see https://github.com/huaweicloud/spring-cloud-huawei/wiki/using-governance

First of all, we should keep the specification language independent. For your requirement, I think you can add this logic when you create your governance request, and set the header you need.